src/sql/azdata.proposed.d.ts

```diff
@@ -446,6 +446,22 @@ declare module 'azdata' {
 		showOnConnectionDialog?: boolean;
 	}
 
+	// Object Explorer interfaces  --------------------------------
+	export interface ObjectExplorerSession {
+		/**
+		 * Authentication token for the current session.
+		 */
+		token?: accounts.AccountSecurityToken | undefined;
+	}
+
+	export interface ExpandNodeInfo {
+		/**
+		 * Authentication token for the current session.
+		 */
+		token?: accounts.AccountSecurityToken | undefined;
```

Comment:
@Charles-Gagnon Is it possible to take this dependency or we shouldn't? I don't see a way of defining it in https://github.com/microsoft/sqlops-dataprotocolclient/blob/main/src/types.ts#L39 unless I introduce the 'accounts' namespace too..

Reply:
I'll resolve this for now, as I need to understand goals for **sqlops-dataprotocolclient**, and whether it should define parity with all types in ADS? Will probably add missing types there once I get clarity on process. I'll take this offline.

Reply:
sqlops-dataprotocolclient references the `@types/azdata` package so that'd be what you'd use to reference the token types from azdata.d.ts.